src/api/spec/models/badge_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Badge, vcr: true do
```

Comment:
```suggestion

RSpec.describe Badge, type: :model do
```
No need to require `webmock/rspec` or use `vcr`. It's a good practice to set the type of the spec, in this case to `:model`.